streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -76,8 +81,9 @@
 
     private final String name;
     private final String parentDir;
+    private final Set<KeyValueIterator> openIterators = new HashSet<>();
 
-    protected File dbDir;
+    File dbDir;
```

Comment:
`dbDir` is referenced in RocksWindowDBStore's segment; and by default it is `package private` not `private` since otherwise the build will break.